AirCasting/SessionViews/StreamsView.swift

```diff
@@ -0,0 +1,67 @@
+// Created by Lunar on 29/06/2021.
+//
+
+import SwiftUI
+
+struct StreamsView: View {
+    
+    @Binding var selectedStream: MeasurementStreamEntity?
+    @ObservedObject var session: SessionEntity
+    var threshold: SensorThreshold
+    @EnvironmentObject private var microphoneManager: MicrophoneManager
+    let showValues: Bool
+    
+    var body: some View {
+        if session.deviceType == .MIC {
+            HStack {
+                measurementsMic
+                Spacer()
+                //This is a temporary solution for stopping mic session recording until we implement proper session edition menu
+                if microphoneManager.session?.uuid == session.uuid, microphoneManager.isRecording && (session.status == .RECORDING || session.status == .DISCONNETCED) {
```

Comment:
as it's temporary I'm not commenting ;)

Reply:
It's not mine, but it'll be changed in the future :)